my-app/src/app/blog/social-media-links/social-media-links.component.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    <a href="https://www.facebook.com/doctoriOS.repair/">
+        <img src="https://www.stlukeseye.com/wp-content/uploads/2018/03/like-us-on-facebook.jpg">
```

Comment:
I like how this is called social media link(S) but it's only 1 lol

ps... that doesn't mean change the name